### DISCUSSION TITLE
improving your package

### DISCUSSION THREAD

**Discussion by hossein-holy** [staudenmeir/laravel-adjacency-list#176] [Ideas]
hi, your package is very useful and very thank you for this package.
I'm have some Ideas for improving your package. you can add checking functions like `isChildren($id, $depth = null)` or `isDescendants($id, $depth = null)`  check if given `id` is in children of object. also can define default parameter `$depth` if need to search in specific Depth.  in the same way `isSiblings($id)`, `isParent($id)`, `isAncestors($id)` and more.

**Comment by lrljoe** [comment]
I know from experience he'll always welcome a PR if you want to code that out!
